db/macc_glue.c

```diff
@@ -1127,11 +1128,14 @@ static int add_cmacc_stmt(dbtable *tbl, int alt, int allow_ull,
             }
 
             schs_indx[nschs_indx] =
-                clone_server_to_client_tag(sch_ondisk->ix[i], cname_buf);
+                clone_server_to_client_tag(idxsch, cname_buf);
             if (!schs_indx[nschs_indx++]) {
                 rc = -1;
                 goto err;
             }
+            /* keys are also in ondisk format */
```

Comment:
get_size_of_schema?

Reply:
that function does exactly the same thing, but it is too generic. We only really need that for dyntags